src/govuk/i18n.mjs

```diff
@@ -263,8 +263,8 @@ I18n.pluralRulesMap = {
  * languages without plurals, such as Chinese.
  * https://unicode-org.github.io/cldr-staging/charts/latest/supplemental/language_plural_rules.html
  *
- * @param    {number}  n  - The `count` number being passed through. This must be a positive integer. Negative numbers and decimals aren't accounted for.
- * @returns  {string}     - The string that needs to be suffixed to the key (without separator).
+ * @param {number} n - The `count` number being passed through. This must be a positive integer. Negative numbers and decimals aren't accounted for.
+ * @returns {string} The string that needs to be suffixed to the key (without separator).
```

Comment:
Not 100% sure that this is the right way to document `I18n.pluralRules` (it's an object containing multiple functions, rather than a function that takes params itself)

(ES Lint doesn't seem to mind… so could look at fixing this later)